WordPress/Classes/ViewRelated/NUX/SignupGoogleViewController.swift

```diff
@@ -69,17 +69,25 @@ extension SignupGoogleViewController: GIDSignInDelegate {
 
         let context = ContextManager.sharedInstance().mainContext
         let service = SignupService(managedObjectContext: context)
-        service.createWPComeUserWithGoogle(token: token, success: { [weak self] in
+        service.createWPComeUserWithGoogle(token: token, success: { [weak self] (accountCreated) in
             SVProgressHUD.dismiss()
-            self?.performSegue(withIdentifier: .showSignupEpilogue, sender: self)
-            WPAnalytics.track(.signupSocialSuccess)
+            if accountCreated {
+                self?.performSegue(withIdentifier: .showSignupEpilogue, sender: self)
+                WPAnalytics.track(.signupSocialSuccess)
+            } else {
+                self?.navigationController?.dismiss(animated: true, completion: nil)
```

Comment:
Instead of dismissing, this should show the *login* epilogue. This will help the user understand what just happened (they logged into an account with these web sites).